core/src/ast_builder/expr/function.rs

```diff
@@ -49,5 +62,14 @@ mod tests {
         let actual = expr("base - 10").abs();
         let expected = "ABS(base - 10)";
         test_expr(actual, expected);
+
+        // FLOOR
+        let actual = floor(col("num"));
+        let expected = "FLOOR(num)";
+        test_expr(actual, expected);
+
+        let actual = expr("base - 10").floor();
+        let expected = "FLOOR(base - 10)";
+        test_expr(actual, expected);
     }
```

Comment:
Instead of explaining that certain tests are about `abs()`, `floor()`, etc using comments(`// ABS`, `// FLOOR`), 
how about we separate them into different test functions?

I can't select the lines from 56~64 at github, so it's kind of hard to explain, but it's kind of like this.

```rust
#[cfg(test)]
mod tests {
    use crate::ast_builder::{abs, floor, col, expr, test_expr};

    #[test]
    fn function_abs() {
        let actual = abs(col("num"));
        let expected = "ABS(num)";
        test_expr(actual, expected);

        let actual = expr("base - 10").abs();
        let expected = "ABS(base - 10)";
        test_expr(actual, expected);
    }

    #[test]
    fn function_floor() {
        let actual = floor(col("num"));
        let expected = "FLOOR(num)";
        test_expr(actual, expected);

        let actual = expr("base - 10").floor();
        let expected = "FLOOR(base - 10)";
        test_expr(actual, expected);
    }
}
```

Reply:
@bugoverdose  Yes. It totally makes sense. Thank you for pointing it out!